src/couch_mrview/src/couch_mrview_debug.erl

```diff
@@ -34,7 +50,100 @@ help(view_signature) ->
     io:format("
     view_signature(ShardName, DDocName)
     --------------
+
     Returns a view signature for given ddoc for a given (non clustered) database.
+
+    ---
+    ", []);
+help(index_state) ->
+    io:format("
+    index_state(Pid)
+    --------------
+
+    Pid: Pid of couch_index:init/1, specifically an mrview index.
+
+    Returns a state map that includes the signature, db_name, idx_name, update_seq,
+    purge_seq, view_file_path, and pending_updates for an index.
+
+    ---
+    ", []);
+help(view_state) ->
+    io:format("
+    view_state(PidOrIdxState)
+    view_state(Pid, ViewName)
+    view_state(IdxState, ViewName)
+    --------------
+
+    PidOrIdxState: Pid of couch_index:init/1, specifically an mrview index, or the state
+    of an mrview index.
+    Pid: Pid of couch_index:init/1, specifically an mrview index.
+    IdxState: State of an mrview index.
+    ViewName: Name of the view to be queried or undefined.
+
+    Returns a state map that includes the id_num, update_seq, purge_seq, reduce_funs, def,
+    btree_size, and options for a ViewName if specified or all views if not.
+
+    ---
+    ", []);
+help(index_view_state) ->
+    io:format("
+    index_view_state(Pid)
+    index_view_state(Pid, ViewName)
+    --------------
+
+    Pid: Pid of couch_index:init/1, specifically an mrview index.
+    ViewName: Name of the view to be queried or undefined.
+
+    Returns a state map that includes the index state returned by index_state/1 and the view
+    state returned by view_state/2. Like view_state/2, a ViewName can be specified or not.
+
+    ---
+    ", []);
+help(index_report) ->
```

Comment:
For all report functions please provide an example of the output (see [example in `couch_index_debug:print_linked_processes/0`](https://github.com/apache/couchdb/blob/main/src/couch_index/src/couch_index_debug.erl#L60:L73).